### PR TITLE
Add mouse events

### PR DESCRIPTION
**Pull request by irenesmith**
This PR relates to [MDN/Sprints #955](https://github.com/mdn/sprints/issues/955) - the refactoring of mouse events.

### CODE REVIEW

File: api/Element.json [hunk 1]
```diff
@@ -4391,6 +4495,422 @@
           }
         }
       },
+      "mousedown_event" {
```
Comment (vinyldarkscratch) [inline api/Element.json]:
```suggestion
      "mousedown_event": {
```

Comment (irenesmith) [reply api/Element.json]: I already made the change and pushed it, but thanks!

File: api/Element.json [hunk 2]
```diff
@@ -4391,6 +4495,422 @@
...
+            }
+          },
+          "status": {
+            "experimental": false,
+            "standard_track": true,
+            "deprecated": false
+          }
+        }
+      },
+      "mousewheel": {
```
Comment (a2sheppy) [inline api/Element.json]: Should be "mousewheel_event"

**Comment by a2sheppy** [comment]
Around line 3767 or so, inside the info for `getElementsByTagNameNS`, is `all_elements_selector`. Inside that is the description `getElementsByTagName(*)`. I'm pretty sure that should be `getElementsByTagNameNS(*)`.

File: api/Element.json [hunk 3]
```diff
@@ -4391,6 +4495,422 @@
...
+        "__compat": {
+          "mdn_url": "https://developer.mozilla.org/docs/Web/API/Element/mousewheel_event",
+          "description": "<code>mousewheel</code> event",
+          "support": {
+            "chrome": {
+              "version_added": true
```
Comment (jpmedley) [inline api/Element.json]: I wasn't able to track down the exact commit, but Chrome Status indicates [this was in 31 on desktop only](https://www.chromestatus.com/features/5388191091654656).

Comment (irenesmith) [reply api/Element.json]: Fixed. Thanks!

File: api/Element.json (hunk 2 above)
Comment (irenesmith) [reply api/Element.json]: mousewheel fixed to mousewheel_event

File: api/Element.json [hunk 4]
```diff
@@ -3764,26 +3764,48 @@
...
         "all_elements_selector": {
           "__compat": {
-            "description": "getElementsByTagName(*)",
+            "description": "getElementsByTagNameNS(*)",
             "support": {
               "chrome": {
                 "version_added": "1"
               },
+              "chrome_android": {
+                "version_added": true
```
Comment (jpmedley) [inline api/Element.json]: This should be `"18"`.

**Pull request merged**